Source/PlayerEx.cpp

```diff
@@ -89,9 +105,8 @@ void PlayerEx::SetPlayerName(char *newName)
 
 int PlayerEx::GetSatelliteCount(map_id objectType)
 {
-	if (!isInited) {
+	if (!isInited)
 		return HFLNOTINITED;
-	}

```

Comment:
Most projects are tending towards always include braces around code blocks, even if they are a single line. In some projects, code scanners being used will warn about single line code blocks that don't have braces around them.

Reply:
I'd been working on an old copy of the code, when I noticed you guys had updated it to this style, I followed suit.  Not sure why it's still showing the "old" way...